pkg/agent/manager/manager_test.go

```diff
@@ -66,7 +62,7 @@ func TestInitializationFailure(t *testing.T) {
 	ca, cakey := createCA(t, clk)
 	baseSVID, baseSVIDKey := createSVID(t, clk, ca, cakey, agentID, 1*time.Hour)
 	cat := fakeagentcatalog.New()
-	cat.SetKeyManager(fakeagentcatalog.KeyManager(memory.New()))
+	cat.SetKeyManager(fakeagentkeymanager.New(t, dir))
```

Comment:
The type of keymanager backing that test case doesn't really matter. I was going to leave it but diverting from the other tests cases would this exact kind of question (i.e, "why memory here instead of disk") so I opted for consistency.